libs/sdk-ui-ext/src/internal/translations/en-US.json

```diff
@@ -49,57 +49,57 @@
         "comment": "",
```

Comment:
Please add `._measure` to the end of `dashboard.bucket.measures_title.pie` at line 47 as well.

Reply:
added